### DISCUSSION TITLE
Clickable pictures in Markdown

### DISCUSSION THREAD

**Discussion by ale-lit** [community/community#18210] [New to GitHub] [answered]
Hello! How can I insert images into Readme.md so that they are not clickable?

<img src="https://avatars.githubusercontent.com/u/93784371?s=60&v=4">

Is there such a way?

**Comment by ale-lit** [comment, marked as answer]
I tried many options and they all didn't work.

**Reply by caioagiani** [reply]
@ale-lit try:

```html
<p align="center">
  <a href="#" target="blank"><img src="https://nestjs.com/img/logo_text.svg" width="320" alt="Nest Logo" /></a>
</p>
```

**Reply by ale-lit** [reply]
Thank for your answer. That's what i'm doing now. But the picture is still a link. Can't you remove the link entirely?

<p align="center">
  <a href="#" target="blank"><img src="https://nestjs.com/img/logo_text.svg" width="320" alt="Nest Logo" /></a>
</p>

**Reply by caioagiani** [reply]
@ale-lit Try this (with "##"):

```md
## Image fixed
<p align="center">
  <a href="#Image fixed"><img src="https://nestjs.com/img/logo_text.svg" width="320" alt="Nest Logo" /></a>
</p>
```

## Image fixed
<p align="center">
  <a href="#Image fixed"><img src="https://nestjs.com/img/logo_text.svg" width="320" alt="Nest Logo" /></a>
</p>

**Reply by ale-lit** [reply]
@caioagiani not bad variant, thank you 👍 

But the picture is still a link. And when you hover over it, the cursor changes to a hand. Why is it implemented this way, if in most cases it is not necessary to make a picture a link?

**Comment by mribeirodantas** [comment]
Yes, there is a way.

```
<picture>
  <source media="" srcset="https://nestjs.com/img/logo_text.svg">
  <img alt="Non-clickable image in GitHub Markdown" src="https://nestjs.com/img/logo_text.svg">
</picture>
```

<picture>
  <source media="" srcset="https://nestjs.com/img/logo_text.svg">
  <img alt="Shows an illustrated sun in light color mode and a moon with stars in dark color mode." src="https://nestjs.com/img/logo_text.svg">
</picture>

**Reply by mribeirodantas** [reply]
You can even show different images based on the viewer theme. Check for more details in the official documentation [here](https://docs.github.com/pt/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#specifying-the-theme-an-image-is-shown-to).
